src/main/java/Assignment2/StudentController.java

```diff
@@ -0,0 +1,120 @@
+package Assignment2;
+
+import java.util.Scanner;
+
+public class StudentController {
+
+  public static void main(String[] args) {
+    boolean ProgramEnd = false;
+    StudentService studentService = new StudentService();
+    studentService.getState(); // get data from repo
+    while (!ProgramEnd) { // loop till user select exit
+      System.out.println("Choose any one among the following");
+      System.out.println("1. Add User details.");
+      System.out.println("2. Display User details.");
+      System.out.println("3. Delete User details");
+      System.out.println("4. Save Us");
+      System.out.println("5. Exit");
+
+      Scanner sc = new Scanner(System.in);
+      String ChoiceString = sc.nextLine().trim();
+      // validating input
+      if (ChoiceString.isEmpty() || !ChoiceString.matches("[0-9]*")
+          || Integer.parseInt(ChoiceString) > 5) {
+        System.out.println("Invalid choice");
+        continue;
+      }
+      int choice = Integer.parseInt(ChoiceString);
+      switch (choice) {
+        case 1: { // add data
+          String name, address, courses, age, rollno;
```

Comment:
move each and every case body into private methods. one method should not be this much long.